src/fetch-mock.js

```diff
@@ -56,6 +57,16 @@ function mockResponse (url, config) {
 	return Promise.resolve(new Response(body, opts));
 }
 
+function toRequest(url, options) {
+	var request;
+	if (Request.prototype.isPrototypeOf(url) && !options) {
+		request = url;
```

Comment:
I actually prefer to just remove the ` && !options` and not worry about the error message.